arithmetization/src/main/java/net/consensys/linea/zktracer/module/mxp/MxpData.java

```diff
@@ -382,12 +383,16 @@ private mxpExecutionPath getMxpExecutionPath() {
 
   public int maxCt() {
     return switch (this.getMxpExecutionPath()) {
-      case TRIVIAL -> 1;
-      case NON_TRIVIAL_BUT_MXPX -> 17;
-      case NON_TRIVIAL -> 4;
+      case TRIVIAL -> 0;
```

Comment:
this should be constant coming from oob/constants.lisp

Reply:
This edit should actually not be in this branch. It ended up here because it was a super small one. Anyway, I'll create the constants for MXP and fix it in a new branch / PR.